Sources/Fuzzilli/Lifting/JavaScriptLifter.swift

```diff
@@ -198,7 +198,16 @@ public class JavaScriptLifter: ComponentBase, Lifter {
                 output = BinaryExpression.new() <> input(0) <> " in " <> input(1)
                 
             case let op as BeginFunctionDefinition:
```

Comment:
I think you can simplify this code a bit, e.g.:
```
var identifiers = instr.innerOutputs.map({ $0.identifier })
if op.hasRestParam, let last = instr.innerOutputs.last {
    identifiers[identifiers.endIndex - 1] = "..." +last.identifier
}
let params = identifiers.joined(separator: ",")``` or something like that
Also no need for `;` at the end or brackets around conditions in an `if` :)

Reply:
Ah cool, haven't really done much swift so good to know!